fragments/k8s/cdk/README.md

```diff
@@ -247,8 +247,8 @@ juju expose kubernetes-worker
 
 In Kubernetes, workloads are declared using pod, service, and ingress
 definitions. An ingress controller is provided to you by default and deployed into
-the [default namespace](https://kubernetes.io/docs/user-guide/namespaces/) of the
-cluster. If one is not available, you may deploy it with:
+the `ingress-nginx-kubernetes-worker` [namespace](https://kubernetes.io/docs/user-guide/namespaces/)
```

Comment:
This name is actually derived from the application name, which defaults to kubernetes-worker. I'm not sure if we should mention this or not, but I thought I would...

Reply:
Added!